app/api/scraping/instagram-enhanced/route.ts

```
@@ -0,0 +1,480 @@
+        if (!keywords || !Array.isArray(keywords) || keywords.length === 0) {
+            log.warn('Enhanced Instagram API validation failed', { 
+                requestId, 
+                reason: 'invalid_keywords',
+                keywords 
+            }, LogCategory.INSTAGRAM);
+            return createErrorResponse('Keywords are required and must be an array', 400, requestId);
+        }
+
+        // Sanitize keywords for AI processing
+        const sanitizedKeywords = keywords.map(keyword => {
+            const sanitized = keyword.replace(/[\u0000-\u001F\u007F-\u009F\uD800-\uDFFF]/g, '');
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)